templates/master/00-master/_base/files/etc-etcd-etcd-conf.yaml

```diff
@@ -0,0 +1,29 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/etcd/etcd.conf"
+contents:
+  inline: |
+    #[member]
+    ETCD_SNAPSHOT_COUNT=100000
+    ETCD_HEARTBEAT_INTERVAL=100
+    ETCD_ELECTION_TIMEOUT=1000
+
+    #[cluster]
```

Comment:
nit this should not exist here.

Reply:
After talking about this I am fine dropping these. In general, I am only explicitly defining defaults. But to eliminate possible confusion I have removed.